dsmr_exporter/dsmr_exporter.py

```diff
@@ -181,13 +181,13 @@ def connect_elastic_output(self, host, port):
             # self.elastic_host = elasticsearch.Elasticsearch([host + ':' + str(port)], sniff_on_start=True)
             self.elastic_host = elasticsearch.Elasticsearch([host + ':' + str(port)])
         except elasticsearch.exceptions.TransportError:
-            self.logger.info("warning: elasticsearch not sniffable, continueing without sniffing")
+            self.logger.info("warning: elasticsearch not sniffable, continuing without sniffing")
             self.elastic_host = elasticsearch.Elasticsearch([host + ':' + str(port)])
         # todo: index template upload
 
     def telegram_to_json(self, telegram):
-        # todo: detectet if input is ok
-        doc = {'@timestamp': datetime.datetime.utcnow()}
+        # todo: detect if input is ok
+        doc = {'@timestamp': datetime.datetime.now(datetime.timezone.utc)}
```

Comment:
Both single and double quotes are used. I don't know if there's a good reason to mix them, but otherwise use one or the other consistently.